MCDataProducts/inc/ProcessCode.hh

```diff
@@ -73,7 +73,7 @@ namespace mu2e {
       G4MinEkineCuts,         G4MaxTimeCuts,           OpAbsorption,        OpBoundary, // 123
       Scintillation,          inelastic,               G4ErrorEnergyLoss,   G4ErrorStepLengthLimit, // 127
       G4ErrorMagFieldLimit,   ePairProd,               FieldPropagator,     Mu2eRecorderProcess,  // 131
-      mu2eProtonInelastic,    RadioactiveDecayBase,
+      mu2eProtonInelastic,    RadioactiveDecayBase,    mu2eLowEnergyGammaKilled,
```

Comment:
This will conflict with already merged PR#313

Reply:
Is the best way to handle this via rebasing against the updated Offline?